File: reverse-string.php
<?php

class Solution {

/**
 * @param String[] $s
 * @return NULL
 */
function reverseString(&$s) {
    $left = 0;
    $right = count($s) - 1;
    $swap = 0;
    print_r($s);
    while( $left < $right ){
        $swap = $s[$right];
        $s[$right] = $s[$left];
        $s[$left] = $swap;
        $left++;
        $right--;
    }
    print_r($s);
}
}

$s = ["h","e","l","l","o"];
$sol = new Solution();
$sol->reverseString($s);

File: reverse-string.py
class Solution:
    def reverseString(self, s: list[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        print(s)
        s = s[::-1]
        print(s)

s = ["h","e","l","l","o"]
sol = Solution()
sol.reverseString(s)